### ISSUE TITLE
LR scheduler

### ISSUE THREAD

**Issue by lunathanael** [lunathanael/racing#4]
Optimizers already support setting learning rates, should add LR schedulers like AnnealingLR.

See: https://docs.pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LRScheduler.html
